EmpireLegends/MainGame.cpp

```diff
@@ -40,3 +42,62 @@ void MainGame::afterAction()
 	this->players->rotate();
 	std::cout << getCurrentPlayer()->getName() << " plays next." << std::endl;
 }
+
+
+void MainGame::maingameloop(vector<Card*> cardVector, int numOfTurns , vector<Player*> myPlayers) {
+       
+    int cardposition = 0;
+    std::cout << myPlayers[0]->getBalance(); 
+    std::cout << myPlayers[1]->getBalance();
```

Comment:
Do we need to display these balances? Also, if we're to display them, maybe we should display all of them, not only the 2 first ones